README.md

```diff
@@ -29,6 +72,12 @@ This will create files with the environment variables needed to run the developm
 This will build and start all of the docker images using docker-compose.
 7. Set `dev.muckrock.com` to point to localhost - `echo "127.0.0.1   dev.muckrock.com" | sudo tee -a /etc/hosts`
 8. Enter `dev.muckrock.com` into your browser - you should see the MuckRock home page.
+9. In  `.envs/.local/.django` (**in the MuckRock project**) set the following environment variables:
```

Comment:
Is it necessary to restart Docker Compose at this point?  There are explicit directions in DocumentCloud to do so after editing the envvars

Reply:
i think it is for the environment variables to take effect, added in line here as well